src/main/resources/assets/purpeille/lang/ko_kr.json

```diff
@@ -0,0 +1,190 @@
+    "advancements.purpeille.ultraviolet_complex_burns.description": "이런, 자외선 복합체에 그만 화상을 입고 말았네요..",
+    "advancements.purpeille.lightning_blocks.title": "하늘 쪼개기",
+    "advancements.purpeille.lightning_blocks.description": "두 가지의 금속 번개 블록을 모두 모으세요",
+    "advancements.purpeille.encased_core.title": "사례 연구",
+    "advancements.purpeille.encased_core.description": "오븐에서 캡슐 코어를 꺼내세요",
+    "advancements.purpeille.ancient_guardian.title": "순교자",
+    "advancements.purpeille.ancient_guardian.description": "고대 가디언으로 당신의 죽음을 방지하세요",
+    "advancements.purpeille.ancient_guardian_deadly.title": "죽음의 선물",
+    "advancements.purpeille.ancient_guardian_deadly.description": "고대 가디언을 사용해 적을 다섯 마리 죽이고, 다른 차원으로 가세요",
+    "advancements.purpeille.vacuous_core.title": "쓸만한 심연",
+    "advancements.purpeille.vacuous_core.description": "캡슐 코어를 공허에 던지세요",
+    "advancements.purpeille.overclocked_core.title": "마침 마력이 부족한 참이었어",
+    "advancements.purpeille.overclocked_core.description": "오버클럭 코어를 만드세요",
+    "advancements.purpeille.ancient_gateway.title": "지켜보고 있다",
+    "advancements.purpeille.ancient_gateway.description": "고대 관문을 사용하세요",
+    "advancements.purpeille.ancient_gateway_far.title": "공허 도약",
+    "advancements.purpeille.ancient_gateway_far.description": "고대 관문을 통해 1만 블럭을 여행하세요",
+    "death.attack.guardianWrath": "%s이(가) %s의 고대 가디언의 분노를 느꼈습니다.",
+    "death.attack.ultravioletComplex": "%s이(가) 윤나는 자외선 복합체에 의해 구워졌습니다.",
+    "effect.purpeille.void_immunity": "공허 피해 면역",
+    "command.purpeille.warpath.invalid.revelation": "'%s'은(는) 올바르지 않은 옵션입니다.",
+    "command.purpeille.warpath.invalid.aspect": "'%s'은(는) 올바르지 않은 추가옵션입니다.",
+    "command.purpeille.warpath.invalid.stack": "%s이(가) 아이템을 들고 있지 않습니다.",
+    "command.purpeille.warpath.success.add": "[%s] 옵션을 성공적으로 추가했습니다.",
+    "command.purpeille.warpath.success.remove": "[%s] 옵션을 성공적으로 제거했습니다.",
+    "tag.purpeille.mining_experience": "채굴 경험치 제공",
+    "tag.purpeille.aspect_catalyst": "추가옵션 부여 촉매",
+    "tag.purpeille.revelation_catalyst": "옵션 부여 촉매",
+    "tag.purpeille.warpath_base": "옵션 부여 아이템",
+    "subtitles.purpeille.ancient_gateway.teleport": "고대 관문이 텔레포트시킴",
+    "subtitles.purpeille.rubble.open": "돌 유물함 열림",
+    "subtitles.purpeille.rubble.close": "돌 유물함 닫힘",
+    "use.purpeille.creative_core": "Set core modifier to %d",
+    "use.purpeille.ancient_guardian.has_tethered_player": "Guardian already has a tethered player",
+    "use.purpeille.ancient_guardian.cooldown": "Cooldown: %d",
+    "message.purpeille.ancient_guardian.void_tether_restored": "Your Void Tether has been restored.",
+    "tooltip.purpeille.creative_core": "Core modifier: ",
+    "rei.purpeille.ancient_oven": "공허 집속",
+    "rei.purpeille.ancient_oven.damage": "%d Damage",
+    "emi.category.purpeille.ancient_oven": "공허 집속",
+    "emi.purpeille.ancient_oven.damage": "%d Damage",
+    "emi.purpeille.chance": "%d%% Chance",
+    "patchouli.purpeille.ancients_almanac.landing_text": "고대의 비밀을 깨우쳐줄 개요서, $(italic)고대 역서$()",

```

Comment:
You should keep this line as English, since the only translation of the guidebook is in English right now